core/contracts/financial-templates/implementation/ExpiringMultiPartyCreator.sol

```diff
@@ -0,0 +1,36 @@
+pragma solidity ^0.5.0;
+
+pragma experimental ABIEncoderV2;
+
+import "../../tokenized-derivative/AddressWhitelist.sol";
+import "../../tokenized-derivative/ContractCreator.sol";
+import "../../common/Testable.sol";
+import "./ExpiringMultiParty.sol";
+
+/**
+@title Expiring Multi Party Contract creator
+@notice Factory contract to create and register new instances of expiring multiparty contracts
+*/
+contract ExpiringMultiPartyCreator is ContractCreator, Testable {
+    constructor(bool _isTest, address _finderAddress) public ContractCreator(_finderAddress) Testable(_isTest) {}
+
+    event CreatedExpiringMultiParty(address expiringMultiPartyAddress, address partyMemberAddress);
+
+    /**
+     * @notice Creates an instance of expiring multi party and registers it within the finder
+     * @dev caller is automatically registered as the first (and only) party member.
+     * @param params is a `ConstructorParams` object from ExpiringMultiParty
+     */
+    function createExpiringMultiParty(ExpiringMultiParty.ConstructorParams memory params) public returns (address) {
+        ExpiringMultiParty derivative = new ExpiringMultiParty(params);
```

Comment:
Should we pass through _isTest and _finderAddress? Because they are set both on the factory and on the created instance.

Reply:
+1